spec/fixtures/concept_collection/concept-3.1.1.1.yaml

```diff
@@ -15,9 +15,8 @@ eng:
       process, etc."
   language_code: eng
   entry_status: valid
-  sources:
-  - type: authoritative
-    origin:
+  authoritativeSource:
+  - origin:
```

Comment:
@strogonoff Also, do we have any sample in-use YAML files that has this `authoritativeSource:` format?   I tried searching it with GitHub to no avail.

Reply:
All glossary repositories managed with Paneron would use it, I don’t know any except https://github.com/paneron/iev-data-subset/ for example. You can obtain a sample by creating a new term using Paneron w/Glossarist, e.g. in some sandbox repository.

Reply:
Fixtures in this repository are outdated. We don’t use top-level `term,` `termid`, `eng` properties, etc. in Glossarist 2 model with separate `concept` and `localized-concept`. But Geolexica build seems to work with latest data, so I don’t know if tests work.

Reply:
Thank you, I think the outdated fixtures confused me.